packages/admin/admin/src/form/FieldContainer.tsx

```diff
@@ -115,16 +126,19 @@ export const FieldContainerComponent: React.FC<WithStyles<typeof styles> & Field
     warning,
     scrollTo = false,
     fieldMargin = "onlyIfNotLast",
+    helperText,
 }) => {
     const hasError = !!error;
     const hasWarning = !!warning;
+    const hasHelperText = !!helperText;
 
     const formControlClasses: string[] = [classes.root];
     if (variant === "vertical") formControlClasses.push(classes.vertical);
     if (variant === "horizontal") formControlClasses.push(classes.horizontal);
     if (fullWidth) formControlClasses.push(classes.fullWidth);
     if (hasError) formControlClasses.push(classes.hasError);
     if (hasWarning && !hasError) formControlClasses.push(classes.hasWarning);
+    if (hasHelperText) formControlClasses.push(classes.helperText);
```

Comment:
I suppose this should be

```suggestion
    if (hasHelperText) formControlClasses.push(classes.hasHelperText);
```

Please verify if the styling still works after the change

Reply:
Styling is verified 👍